### PR TITLE
BZ1854383 Release Notes update

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1854383

### CODE REVIEW

**Comment by bmcelvee** [comment]
@lihongan would you mind reviewing this release notes update, please? Thanks!

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -1860,6 +1860,8 @@ fail to boot. Do not select this option.
 * You might receive a message that the system is unable to mount a volume when the volume contains a large number of files. This can happen when a Pod mounts a volume that is set with `FSGroup SecurityContext` because the GID ownership of the files must be recursively updated for all files on the volume. Users should expect that Pods using volumes with a large number of files and the `FSGroup SecurityContext` setting can take considerable time to start.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1515907[*BZ#1515907*])
 
+* On Azure when upgrading from 4.4 to 4.5, the Ingress Operator fails to ensure a DNSRecord due to errors refreshing the token. Restarting the Ingress Operator fixes the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1854383[*BZ#1854383*])
```
Comment (lihongan) [inline release_notes/ocp-4-5-release-notes.adoc:1863]: `the Ingress Operator might fails...` since it happens randomly and rarely.

Comment (lihongan) [reply release_notes/ocp-4-5-release-notes.adoc:1863]: other looks good. thank you @bmcelvee

Comment (bmcelvee) [reply release_notes/ocp-4-5-release-notes.adoc]: Thank you!

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review. Thanks!

File: release_notes/ocp-4-5-release-notes.adoc [hunk 2]
```diff
@@ -1867,6 +1867,8 @@ frequently enough, systemd has trouble reaping all of its new children, and some
 conmon processes can become zombies.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1852064[*BZ#1852064*])
 
+* On Azure when upgrading from 4.4 to 4.5, the Ingress Operator can fail to ensure a DNSRecord due to errors refreshing the token. Restarting the Ingress Operator fixes the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1854383[*BZ#1854383*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-5-release-notes.adoc:1870]: s/Azure/Microsoft Azure

**Comment by ahardin-rh** [comment]
@bmcelvee Just one small suggestion from me. Otherwise, looks good!

**Pull request merged**